src/Console/ConfigurationResolver.php

```diff
@@ -253,7 +255,21 @@ public function getConfigFile()
     public function getDiffer()
     {
         if (null === $this->differ) {
-            $this->differ = false === $this->options['diff'] ? new NullDiffer() : new SebastianBergmannDiffer();
+            if (false !== $this->options['differ']) {
+                // 'diff' value does not matter here
```

Comment:
If `--differ` option is passed a value then the tool behaves as if `--diff` was passed as well, even if it wasn't.